api/src/main/java/io/kafbat/ui/serdes/builtin/ProtobufFileSerde.java

```diff
@@ -426,6 +427,27 @@ private Map<String, ProtoFile> loadFilesWithLocations() {
       }
       return filesByLocations;
     }
+
+    /**
```

Comment:
Do we keep this until the issue is resolved on wire side? Or is this a different issue?

Reply:
I am not sure what issue you are referring to

I noticed this problem locally when I ran these tests manually

Reply:
Sorry I thought we've discussed this with you. See #711 and https://github.com/kafbat/kafka-ui/pull/712#discussion_r1885852375

This approach is more clean and I don't mind accepting it as a temporarily solution until the bug is fixed in wire.

Reply:
Ah, I see. Thanks for the context

When you say "bug in wire," does that refer to an eternal bug report you can point me to? Or an internal module that needs attention?

Either way, this temporary fix would be nice to allow running under Windows

Reply:
Wire bug relates to this: https://github.com/kafbat/kafka-ui/pull/712#issuecomment-2543916366 -> https://github.com/square/wire/pull/3215